bokehjs/src/coffee/callback/customjs.coffee

```diff
@@ -1,5 +1,5 @@
 _ = require "underscore"
-Model = require "../models/model"
+Model = require "../model/model"
```

Comment:
AFAIK we seem to use singular directory names in BokehJS. In Bokeh, most of what are directories in BokehJS (i.e. renderer/glyph/) is a single file with a plural name (glyphs.py) in Bokeh.

I was just trying to continue that pattern.